data/iteration-summary.json

```diff
@@ -0,0 +1,398 @@
+        "from": "2020-01-01",
+        "to": "2020-01-01",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
+    {
+        "from": "2020-02-01",
+        "to": "2020-02-01",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
+    {
+        "from": "2020-02-15",
+        "to": "2020-02-15",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
+    {
+        "from": "2020-03-03",
+        "to": "2020-03-03",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
+    {
+        "from": "2020-03-03",
+        "to": "2020-03-03",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
+    {
+        "from": "2020-03-30",
+        "to": "2020-03-30",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
+    {
+        "from": "2020-04-11",
+        "to": "2020-04-11",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
+    {
+        "from": "2020-04-20",
+        "to": "2020-04-20",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 1.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
+    {
+        "from": "2020-05-04",
+        "to": "2020-05-04",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
+    {
+        "from": "2020-05-16",
+        "to": "2020-05-16",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
+    {
+        "from": "2020-05-30",
+        "to": "2020-05-30",
+        "dev": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        },
+        "design": {
+            "story_points": 0.0,
+            "issues_closed": 0.0
+        }
+    },
```

Comment:
All numbers above here are zero because I set the start date to April 1, 2020

Reply:
I would like to have more historical data (... I'll have to figure out how far back, I guess. I can look into that). The problem with doing this now is the current scripts recalculate everything, right? If we update the scripts to only find and calculate since the data files were last updated, will that make things manageable?